From 489a38bcabb14d5efb209e80e2dd6e0526753f6f Mon Sep 17 00:00:00 2001
From: Erlend <me@erlend.io>
Date: Wed, 30 Sep 2020 14:09:24 +0200
Subject: [PATCH] add hint to make fedora find the qmlplugindump

---
 src/QmlPlugins/QmlPluginsConfig.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/QmlPlugins/QmlPluginsConfig.cmake b/src/QmlPlugins/QmlPluginsConfig.cmake
index d7b2876..c700838 100644
--- a/src/QmlPlugins/QmlPluginsConfig.cmake
+++ b/src/QmlPlugins/QmlPluginsConfig.cmake
@@ -2,7 +2,7 @@
 # target before loading this plugin.
 
 if(NOT TARGET qmlplugindump)
-    find_program(qmlplugindump_exe qmlplugindump)
+    find_program(qmlplugindump_exe qmlplugindump HINTS /usr/lib64/qt5/bin)
 
     if(NOT qmlplugindump_exe)
       message(FATAL_ERROR "Could not locate qmlplugindump.")
-- 
2.26.2

